setup.py

```diff
@@ -112,6 +112,7 @@
         "tqdm>=4.62.3,<5.0",
         "traitlets>=5.3.0",
         "watchdog>=3.0,<4",
+        "tomli>=2.0.1",
```

Comment:
This is how you would specify backports
```suggestion
        "tomli>=2.0.1 ; python_version<3.11",
```